sdk/cosmosdb/cosmos/ci.yml

```diff
@@ -15,3 +15,7 @@ jobs:
   - template: ../../../eng/pipelines/templates/jobs/archetype-sdk-client.yml
     parameters:
       PackageName: "@azure/cosmos"
+
+  - template: ../../../eng/pipelines/templates/jobs/node6-single-client.yml
```

Comment:
Does this need to be in the CI vs the tests.yml?

Reply:
Can you tell me more about what tests.yml does? I don't see one in the repo.

Reply:
Currently it is https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/cosmosdb/cosmos/.azure-pipelines/cosmos.test.public.emulator.yml because we haven't converted the tests in this repo into a tests.yml file yet.

Eventually the plan is that tests.yml will be the nightly/live test runs for a given service. Where as ci.yml will be the basic testing we do during PR validation so the question is should node6 be part of the PR validation matrix or just the nightly test runs?

Reply:
My understanding is that they want it for PR validation. The service team plans to support Node 6 for at least a while longer until the next major release of the library, and after that they plan to support the deprecated branch for at least an additional year.

Reply:
OK if they want it for CI then this is good. We may want to put test coverage in both however at some point.